devops/pre_commit.exclude_filter

```diff
@@ -0,0 +1,5 @@
+buffer
```

Comment:
How does this exclude_filter work?  If there are individual CTS tests we want to enable/disable, what is the syntax and best approach for that?